lib/Mailer/Methods/MailPoet.php

```diff
@@ -37,6 +39,14 @@ function send($newsletter, $subscriber, $extra_params = []) {
       return Mailer::formatMailerErrorResult($this->error_mapper->getInvalidApiKeyError());
     }
 
+    $subscribers_for_blacklist_check = is_array($subscriber) ? $subscriber : [$subscriber];
+    foreach ($subscribers_for_blacklist_check as $sub) {
+      if ($this->isBlacklisted($sub)) {
+        $error = $this->error_mapper->getBlacklistError($sub);
+        return Mailer::formatMailerErrorResult($error);
```

Comment:
Does it mean in this case we stop the whole batch?

Reply:
Yes, see Slack discussion: https://mailpoet.slack.com/archives/C02MTKAJL/p1562749734058000